prow/deploy.sh

```diff
@@ -69,5 +83,9 @@ trap restore-context EXIT
 ensure-context k8s-prow-builds us-central1-f prow
 ensure-context k8s-prow us-central1-f prow
 echo " $(color-green done), Deploying prow..."
+for s in {5..1}; do
```

Comment:
what are we waiting on here? can we explicitly wait for it instead of hoping we've slept enough?

Reply:
ok that SGTM, I'm always concerned when I see a `sleep` call because that usually means we need something to be ready and we're just hoping that waiting some arbitrary amount of time will work.

waiting for a human obviously needs an arbitrary amount of time :^)